Movies-Selection/Movies-Selection/DataBase/DetailFilmRepository/DescriptionFilmRepository.swift

```diff
@@ -0,0 +1,52 @@
+// DescriptionFilmRepository.swift
+// Copyright © RoadMap. All rights reserved.
+
+import CoreData
+import Foundation
+/// Протокол DataStarage
+protocol DescriptionFilmDataBaseProtocol {
+    func getDescriptionFromDataStorage() -> [DataDescriptionDetailsFilm]
+    func saveMovies(movies: DescriptionDetailsFilm)
+    func deleteAllData(_ entity: String)
+}
+
+/// Репозиторий для 2 модуля
+final class DescriptionFilmRepository: DescriptionFilmDataBaseProtocol {
```

Comment:
```suggestion
final class DescriptionFilmRepository: DescriptionFilmDataBaseProtocol {
```
1 н универсальный лучше